### PR TITLE
Update upgrade-service-applications-to-sharepoint-server-2016.md

### PR DESCRIPTION
**Pull request by tseward**
Added clarity around what steps only applied to a Custom role.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/upgrade-and-update/upgrade-service-applications-to-sharepoint-server-2016.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ To upgrade a service application database, you create a new service application
 > Word Automation Services and Machine Translation Services can't be upgraded. A new service instance will need to be created. 
   
 > [!IMPORTANT]
-> The following steps only apply to the Custom server role type. For more information on server role types, see [Planning for a MinRole server deployment in SharePoint Server 2016](../install/planning-for-a-minrole-server-deployment-in-sharepoint-server.md). 
+> The following steps outlining starting service instances only applies to the Custom server role type. For more information on server role types, see [Planning for a MinRole server deployment in SharePoint Server 2016](../install/planning-for-a-minrole-server-deployment-in-sharepoint-server.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/upgrade-and-update/upgrade-service-applications-to-sharepoint-server-2016.md]:
```suggestion
> The following steps outlining starting service instances only apply to the Custom server role type. For more information on server role types, see [Planning for a MinRole server deployment in SharePoint Server 2016](../install/planning-for-a-minrole-server-deployment-in-sharepoint-server.md). 
```

Comment (nenonix) [inline SharePoint/SharePointServer/upgrade-and-update/upgrade-service-applications-to-sharepoint-server-2016.md]:
```suggestion
> The following procedure for starting service instances only applies to the Custom server role type. For more information on server role types, see [Planning for a MinRole server deployment in SharePoint Server 2016](../install/planning-for-a-minrole-server-deployment-in-sharepoint-server.md). 
```

**Comment by scanum** [comment]
@Techwriter40 Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**